### DISCUSSION TITLE
Problème dead Screen and revive (et yarn run prisma db seed error)

### DISCUSSION THREAD

**Discussion by nibor51** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#321] [Q&A] [answered]
Hello, j'ouvre cette discutions suite au message d'un des joueurs de SOZ

> Update : j'ai enfin testé et y'a un problème d'URI invalide au niveau de soz-props-politic, soz-props-pride et soz-props-radio qui empêche les clients de se co.
> Les annonces type Twitch news qui marchent pô et des soucis de gens qui sont coma mais pas trop. Genre barre de vie à 0, mouvement impossible mais pas considéré coma.

J'ai update mon local, supprimer les contenues de cache client/server et aussi des node_module et drop la database avant de la recréer.
Bon un soucis avec `yarn run prisma db seed`, qui ducoup casse beaucoup de choses, voir le fichier de log joint si quelqu’un peut m’aider la dessus.
[yarn-run-prisma-db-seed.log](https://github.com/user-attachments/files/16120502/yarn-run-prisma-db-seed.log)

Mais aucun problème sur les nouveaux props ou les annonces TN pour m'a par.
Par contre pour les coma oui.

En effet, lors de la mise coma du perso l'écran permettent d'indiquer le motif n'est pas affiché, du-coup pas de cooldown pour la réapparition via UHU entre autres. Le revive via le menu admin fonctionne une première fois, mais si l'on retombe dans le coma il n'est pas possible de s'auto-revive une seconde fois.
Rien concernent se soucis dans les log client ou serveur. J'ai pas encore check le code qui gérer cela donc je ne sais pas d’où vient le soucis, j’ai fix ça sur mon serveur de test en auto-revive chaque personne tombant coma en attendent de me pencher dessus.

**Comment by Laikker** [comment]
Pour le prisma db seed, Passe sur une version supérieur de swc core, dans resources\[soz]\soz-core\package.json
"@swc/core": "1.6.13",

Pour les problème de coma, je reproduit pas.

**Reply by nibor51** [reply]
Me doutait que c'étais un problème de maj pour prisma.

Je retest pour le coma après avoir fix ma DB, mais me semble que javais déjà eu le soucis sur le build précédent. Je checkrai le code dans la nuit pour voir d’où cela peut venir.

**Comment by nibor51** [comment, marked as answer]
le problème vient si on désactive totalement l'utilisation d'un mumble même natif.
dans la function onDeath le `await this.voipService.mutePlayer(true);` bloque donc tous.

donc pour palier au soucis de death screen ne venant pas, il faut que dans le `env.cfg` la ligne `set mumble_disableServer` soit à `false`.
Permettent ainsi de na pas bloquer en attendent un mute du player qui ne se ferra jamais.

![image](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/88200918/95f26e88-8c9c-4f4a-be6b-c77eb65dfb27)
